acceptance/tests/ssl/crl_retrieval_and_replacement.rb

```diff
@@ -0,0 +1,44 @@
+test_name "Crl retrieval and replacement from master" do
+
+  tag 'risk:medium',
+      'server'
+
+  with_puppet_running_on(master, {}) do
+    agents.each do |agent|
+      ca_crl_path = on(master, puppet('config print cadir --section master')).stdout.chomp + "/ca_crl.pem"
+      crl_path = on(agent, puppet('config print ssldir --section agent')).stdout.chomp + "/crl.pem"
```

Comment:
Not blocking this PR, but you could use `puppet config print cacrl --section master` and `puppet config print hostcrl --section master` respectively.

Reply:
Unrelated to this PR, but another thing this reminded me of: when we switch the CA location to being configured by puppetserver, this and tests like it will no longer work. (see https://tickets.puppetlabs.com/browse/SERVER-2225) Just food for thought... we'll need to think about how we want to resolve that.

Reply:
@joshcooper I'll update that and check my acceptance tests pass.